### PR TITLE
Make port configurable

### PR DESCRIPTION
**Pull request by dhawal55**
#92

### CODE REVIEW

**Comment by thebsdbox** [comment]
Awesome ! 

Thanks for this :-)

**Comment by thebsdbox** [comment]
Can you modify this to merge into the 0.2.1 branch?

**Comment by dhawal55** [comment]
Updated to merge into 02.1 branch

**Comment by thebsdbox** [comment]
I’ll take it for a whirl tomorrow and if all good have it merged!! 😀

File: pkg/cluster/clusterLeader.go [hunk 1]
```diff
@@ -65,9 +65,7 @@ func NewManager(path string, inCluster bool) (*Manager, error) {
 			return nil, err
 		}
 
-		// TODO - we need to make the host/port configurable
-
-		config.Host = fmt.Sprintf("%s:6443", id)
+		config.Host = fmt.Sprintf("%s:%v", id, port)
```
Comment (thebsdbox) [inline pkg/cluster/clusterLeader.go:68]: Should this be `%d` for the port?

Comment (dhawal55) [reply pkg/cluster/clusterLeader.go:68]: %v is the value in the default format so it works fine

**Comment by dhawal55** [comment]
FYI, I have tested this and I have kube-vip running on 443

**Comment by thebsdbox** [comment]
`/lgtm` :-)

I'll create a `0.2.1` image soon for testing

**Pull request merged**